presto-main/src/main/java/io/prestosql/sql/planner/planprinter/IoPlanPrinter.java

```diff
@@ -201,17 +203,17 @@ private EstimatedStatsAndCost getEstimatedStatsAndCost()
         public static class TableColumnInfo
         {
             private final CatalogSchemaTableName table;
-            private final Set<ColumnConstraint> columnConstraints;
+            private final Set<ColumnConstraint> columns;
             private final EstimatedStatsAndCost estimate;
 
             @JsonCreator
             public TableColumnInfo(
                     @JsonProperty("table") CatalogSchemaTableName table,
-                    @JsonProperty("columnConstraints") Set<ColumnConstraint> columnConstraints,
+                    @JsonProperty("columns") Set<ColumnConstraint> columns,
```

Comment:
Why is `columnConstraints` being renamed to `columns`? In particular, changing this in the JSON representation will break backward compatibility.

Reply:
I renamed it based on feedback you have me here: https://github.com/prestosql/presto/pull/2743#issuecomment-592796177. Maybe I misunderstood? I'm happy to change it back if you think that's better.

Reply:
Ah, sorry if that was confusing... I may have crossed the names. My overall point was that we shouldn't rename it from what it was (and also not add a new field for the filters, but reuse the existing one).

Reply:
I've changed the element name back to `columnConstraints`